meetup/models.py

```diff
@@ -1,3 +1,62 @@
+from uuid import uuid4
+
+from django.contrib.auth.models import User
```

Comment:
```python
from django.contrib.auth import get_user_model
User = get_user_model()
```
`get_user_model()` provide more flexibility.

Reply:
https://docs.djangoproject.com/en/1.11/topics/auth/customizing/#referencing-the-user-model

👍